src/test/java/net/minecraftforge/test/ItemCanDestroyBlocksInCreativeTest.java

```diff
@@ -0,0 +1,31 @@
+package net.minecraftforge.test;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+
+@Mod(modid = ItemCanDestroyBlocksInCreativeTest.MODID, name = "Item.canDestroyBlockInCreative() Test", version = "1.0", acceptableRemoteVersions = "*")
+public class ItemCanDestroyBlocksInCreativeTest{
+	public static final boolean ENABLE = false;
```

Comment:
Eww... Use 4 spaces instead of tabs; braces should be on new lines, and `if (ENABLE)` spacing, etc.